c/polyjuice.h

```diff
@@ -1010,6 +1018,70 @@ int handle_transfer(gw_context_t* ctx,
   return 0;
 }
 
+int handle_native_token_transfer(gw_context_t* ctx, uint32_t from_id,
+                                 uint256_t value, gw_reg_addr_t* from_addr) {
+  if (g_creator_account_id == UINT32_MAX) {
+    ckb_debug("[handle_native_token_transfer] g_creator_account_id wasn't set.");
+    return ERROR_NATIVE_TOKEN_TRANSFER;
+  }
+  if (!g_eoa_transfer_flag) {
+    ckb_debug("[handle_native_token_transfer] not a native transfer tx");
+    return ERROR_NATIVE_TOKEN_TRANSFER;
+  }
```

Comment:
`g_eoa_transfer_flag` was already checked at
https://github.com/nervosnetwork/godwoken-polyjuice/pull/173/files#diff-8a43ccc334a831d56b5bdfa4712228904f3823f6ec2b82941664ca849ab496b0R1474. We can omit this check.